benches/search_comparison.rs

```diff
@@ -263,7 +264,13 @@ where
     T: TryFrom<usize>,
     <T as TryFrom<usize>>::Error: core::fmt::Debug,
 {
-    let mut seed = 0usize;
+    static SEED: AtomicUsize = AtomicUsize::new(0);
+    let mut seed = SEED
+        .fetch_update(Ordering::SeqCst, Ordering::SeqCst, |value| {
+            Some(value.wrapping_mul(1664525).wrapping_add(1013904223))
+        })
```

Comment:
I actually think I would just `fetch_add(1)` here instead. Otherwise, since we're using the same logic to update the seed as we're doing to generate numbers, we'll just end up with all the random numbers being offset by one step across benchmarks. Whereas adding 1 to the seed will lead to a completely different number series.

Reply:
done